src/utils.rs

```diff
@@ -4,19 +4,6 @@
 
 use ffi;
 use translate::*;
-use UserDirectory;
-
-pub fn get_application_name() -> Option<String> {
```

Comment:
I would (for now at least) just re-export the functions as they were before from src/utils.rs, e.g. `pub use auto::functions::get_application_name` in there.